src/main/java/nextstep/subway/domain/Line.java

```diff
@@ -13,6 +14,9 @@
     private String name;
     private String color;
 
+    @OneToMany(mappedBy = "line", cascade = {CascadeType.PERSIST, CascadeType.MERGE}, orphanRemoval = true)
+    private List<Section> sections = new ArrayList<>();
+
     public Line() {
```

Comment:
```suggestion
    protected Line() {
```
JPA 때문에 기본생성자를 막을 수 없지만 최대한 접근제어자로 객체를 보호해보시기 바랍니다.